blob/src/main/java/com/bazaarvoice/emodb/blob/BlobStoreModule.java

```diff
@@ -339,13 +339,13 @@ CacheRegistry provideCacheRegistry(CacheRegistry cacheRegistry) {
     @Provides @Singleton @TableChangesEnabled
     ValueStore<Boolean> provideTableChangesEnabled(@BlobStoreZooKeeper CuratorFramework curator,
                                                    LifeCycleRegistry lifeCycle) {
-        return lifeCycle.manage(new ZkValueStore<>(curator, "settings/table-changes-enabled", new ZkBooleanSerializer(), true));
+        return lifeCycle.manage(new ZkValueStore<>(curator, "/settings/table-changes-enabled", new ZkBooleanSerializer(), true));
     }
 
     @Provides @Singleton @Maintenance
     MapStore<Double> provideRateLimiterSettings(@BlobStoreZooKeeper CuratorFramework curator,
                                                 LifeCycleRegistry lifeCycle) {
-        return lifeCycle.manage(new ZkMapStore<>(curator, "maintenance-rate-limits", new ZkDoubleSerializer()));
+        return lifeCycle.manage(new ZkMapStore<>(curator, "/maintenance-rate-limits", new ZkDoubleSerializer()));
```

Comment:
ZK path requires leading `/` in new versions.

Reply:
yeah, but prefer to stay in 3.4.x versions (3.4.13 is latest)

Reply:
as zookeeper version was downgraded, should this to be reverted also?

Reply:
I believe it is actually curator, not ZK that enforces this. Additionally, I feel like it may be better to not rely on a leading `/` to be added for us anyway. If you really feel strongly about it than I can experiment with removing it.